Dockerfile

```diff
@@ -14,16 +16,21 @@ RUN set -ex; \
       jq \
       groff \
       py-pip \
-      python \
+      python3 \
+      chromium \
+      udev \
+      ttf-freefont \
       nodejs \
       npm; \
     npm install -g npm;
 
-# Install aws-shell (which also install aws-cli)
-RUN pip install --upgrade \
+# Install aws-shell (which also installs aws-cli) and some dependencies
+RUN pip3 install --upgrade \
       pip \
       aws-shell \
-      awsebcli;
+      awsebcli \
+      boto==2.49.0 \
+      pyppeteer==0.0.25
```

Comment:
Is there a semicolon required here?

Reply:
It's not required, the image builds just fine